### DISCUSSION TITLE
Fix failed integrity files

### DISCUSSION THREAD

**Discussion by Martinligabue** [nextcloud/all-in-one#5588] [Questions] [answered]
I couldn't find a way in the readme or in the discussions to automatically restore files that didn't pass the integrity check.
According to https://docs.nextcloud.com/server/27/admin_manual/issues/code_signing.html I should "upload the correct file and delete the extra ones", but I don't know where the original files are and how to upload them.
I also found this comment: https://help.nextcloud.com/t/some-files-have-not-passed-the-integrity-check/166162/3 but it didn't fix the problem, as it didn't with the last three updates.

**Comment by szaimen** [comment, marked as answer]
Hi, please create a new thread for this at https://help.nextcloud.com. Thanks!
